modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/PainlessLexer.java

```diff
@@ -1,13 +1,17 @@
 // ANTLR GENERATED CODE: DO NOT EDIT
 package org.elasticsearch.painless.antlr;
-import org.antlr.v4.runtime.Lexer;
```

Comment:
Are the changes to this file due to intellij?  What happens if you regen?

Reply:
Picked up in optimize imports. I'll fix this.